### ISSUE TITLE
Using a different set of points

### ISSUE THREAD

**Issue by beraldofilippo** [chili-epfl/attention-tracker#17]
Hi, I'm interested in evaluating how this algorithm performs using a different set of 3D points of the head, for instance using a female head shape to track a female individual. Any advices on that?

Also, how did you manage to get those 3D points? I've looked at the link suggested (wikipedia...) but I could not find anything useful, thanks!

**Comment by alexis-jacq** [comment]
Actually, the points we fund on wikipedia (position of the sellion etc...) are just used to compute the position in space of a head that is already detected. In order to detect the face, we use the face-detection algorithm provided by [DLib](http://dlib.net/). It is based on the recognition of 68 landmarkers of a human face. If you want to just detect women heads, I am not aware of the state of the art, maybe good results are know obtained with convNets ?

**Comment by beraldofilippo** [comment]
Thanks for the response, I should have been more clear though :) sorry!
I can successfully reproduce all of the steps of the algorithm (and actually I've made an Android version of GAZR) I was wondering how to evaluate the performance on a different set of points.

So no need to 'track female shapes', I'm just wondering if changing the set of 3D points according to the individual could give better results (say, if the subject is a caucasian male use a specific  set with anthropometric mean value for caucasians, if the subject is a child use a set with anthropometric mean values for children and so on)

Thanks.
(Still no clue about where to find those 3D points sets :/)

**Comment by alexis-jacq** [comment]
Ah ok ! 
So you want to say "with hands" to the computer that the subject is an adult woman or a child, and then the computer uses specific points for the head pose estimation. That's a good idea, and we are interested (since we use this algorithm for Child-Robot interaction studies) if you find such a set of coordinates! Indeed in https://en.wikipedia.org/wiki/Human_head we just have values for adult men and women but not the children

**Comment by beraldofilippo** [comment]
Exactly. Maybe we should keep this issue open and somebody may help in this.
Thanks.
